### PR TITLE
Rfid rolls

### PR DESCRIPTION
**Pull request by billyeh**
Not sure how well this follows #266 but added an RFID roll view and some buttons on that page for flipping rolls and removing tardies in the case that we're dismissed too late to swipe within the 15 minute window.

### CODE REVIEW

File: ap/attendance/views.py [hunk 1]
```diff
@@ -333,3 +342,58 @@ class AllAttendanceViewSet(BulkModelViewSet):
...
+
+  return HttpResponse('Roll finalized')
+
+@group_required(('attendance_monitors',))
+def rfid_tardy(request, event_id, event_date):
+  event = get_object_or_404(Event, pk=event_id)
+  date = datetime.strptime(event_date, "%Y-%m-%d").date()
+  if not event.monitor == 'RF':
+    return HttpResponse('No event found')
+  rolls = event.roll_set.filter(date=date, status='T')
```
Comment (jontey) [inline ap/attendance/views.py]: could you do `event.roll_set.filter(date=date, status='T').delete() ` instead?

Comment (jontey) [inline ap/attendance/views.py]: Also need to check if a leaveslip is attached to the roll before deleting.... Perhaps override the delete method in the model?

File: ap/schedules/models.py [hunk 2]
```diff
@@ -123,15 +117,7 @@ def date_for_week(self, week):
 
   # checks for time conflicts between events. Returns True if conflict exists.
   def check_time_conflict(self, event):
-    return (self.end >= event.start) and (event.end >= self.start)
-
-  # gets the week from an absolute date of the current term.
-  def week_from_date(self, date):
```
Comment (jontey) [inline ap/schedules/models.py]: Please add back these two functions. Thanks

Comment (billyeh) [inline ap/schedules/models.py]: OK. I don't think we need a non-static method, since the static method will work when called with an instance too.

Comment (jontey) [inline ap/schedules/models.py]: True.. just keep one of them then

File: ap/attendance/views.py (hunk 1 above)
Comment (billyeh) [inline ap/attendance/views.py]: First comment is correct. I was running into some weird bug before but it seems all right.

I don't think we want to override the delete method that way, especially with what this says http://stackoverflow.com/questions/1534986/how-do-i-override-delete-on-a-model-and-have-it-still-work-with-related-delete. I just added the filters.

**Pull request merged**